apps/dolly-frontend/src/main/js/src/components/bestillingsveileder/stegVelger/steg/steg3/IdentVelger.tsx

```diff
@@ -78,8 +81,17 @@ export const IdentVelger = ({ formikBag }: Form) => {
 					{ label: 'NAV syntetisk', value: IdentType.SYNTETISK },
 				]}
 				checked={type}
-				onChange={(e) => handleIdentTypeChange(e.target.value)}
+				onChange={(e) =>
+					handleIdentTypeChange((e.target as HTMLTextAreaElement).value as IdentType)
+				}
 			/>
+
+			{erArenaBestilling && type === IdentType.SYNTETISK && (
+				<AlertStripeAdvarsel>
+					Syntetisk ident er foreløpig ikke støttet i Arena, velg standard ident for gyldig
```

Comment:
Burde kanskje endre teksten til _"...ikke støttet for Arbeidsytelser (Arena)..."_ siden vi ikke bruker Arena som overskrift i bestilling lenger ;)

Reply:
Ja god ide 😄